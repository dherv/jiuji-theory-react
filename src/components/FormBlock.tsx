import styled from 'styled-components';

const FormBlock = styled.div`
  display: flex;
  flex-direction: column;
  width: 100%;
  margin-bottom: 1.5rem;
`;
export default FormBlock;
