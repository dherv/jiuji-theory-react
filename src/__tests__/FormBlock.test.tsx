import 'jest-styled-components';
import React from 'react';
import { render } from '@testing-library/react';
import FormBlock from '../components/FormBlock';
import { FormBlockProps as props } from '../samples/FormBlock.sample';

test('should render FormBlock', () => {
  const { asFragment } = render(<FormBlock {...props} />);
  expect(asFragment()).toMatchSnapshot();
});
