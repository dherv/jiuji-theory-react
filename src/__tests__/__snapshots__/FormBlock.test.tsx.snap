// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormBlock 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  margin-bottom: 1.5rem;
}

<div
    class="c0"
  />
</DocumentFragment>
`;
